src/RolloutScorer/RolloutScorer.Service/Startup.cs

```diff
@@ -0,0 +1,40 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace RolloutScorer.Service
+{
+    public class Startup
+    {
+        // This method gets called by the runtime. Use this method to add services to the container.
```

Comment:
Boilerplate code that will need to be cleaned up and made into a proper SF service.